mava/wrappers/jumanji.py

```diff
@@ -245,18 +255,9 @@ def create_agents_view(grid: chex.Array) -> chex.Array:
             agents_view = jnp.stack(
                 (positions, targets, paths, position_per_agent, target_per_agent), -1
             )
+            # agents_view = jnp.repeat(agents_view, self.num_agents, axis=0)
             return agents_view
 
-        def aggregate_rewards(
-            timestep: TimeStep,
-        ) -> TimeStep[Observation]:
-            """Aggregate individual rewards and discounts across agents."""
-            team_reward = jnp.sum(timestep.reward)
-            reward = jnp.repeat(team_reward, self.num_agents)
-            return timestep.replace(reward=reward)
-
-        timestep = aggregate_rewards(timestep)
-
```

Comment:
If we were aggregating before, we should probably be aggregating again now to keep the logic consistent. Unless Jumanji now does something differently?

Reply:
Yeah Jumanji already returns the sum of rewards and the diffrence between Connector and Maconnector is the individual reward the observation or the thing that @sash-a added for `switch_perspective` [function](https://github.com/sash-a/jumanji/blob/f102941b0b5eb4302dbe624048617803a07fe87a/jumanji/environments/routing/connector/utils.py#L182)

Reply:
@sash-a probably need some help on deciding whether or not to discard this utility since the paper version uses it and if not should it be included in mava only?